web/react/components/file_attachment.jsx

```diff
@@ -100,15 +162,71 @@ export default class FileAttachment extends React.Component {
         var fileUrl = utils.getFileUrl(filename);
         var type = utils.getFileType(fileInfo.ext);
 
-        var thumbnail;
-        if (type === 'image') {
-            thumbnail = (
+        var playbackControls = '';
+        var loadedFile = '';
+        var loadingIndicator = '';
+        if (this.state.mime === 'image/gif') {
+            playbackControls = (
                 <div
-                    ref={filename}
-                    className='post__load'
-                    style={{backgroundImage: 'url(/static/images/load.gif)'}}
+                    className='file-playback-controls play'
+                    onClick={(e) => this.playGif(e, filename)}
+                >
+                    {"►"}
+                </div>
+            );
+        }
+        if (this.state.playing) {
+            loadedFile = (
+                <img
+                    className={'file__loaded ' + this.state.format}
+                    src={fileUrl}
+                />
+            );
+            playbackControls = (
+                <div
+                    className='file-playback-controls stop'
+                    onClick={(e) => this.stopGif(e, filename)}
+                >
+                    {"◼"}
```

Comment:
Chrome on Windows doesn't support this character for some reason even though it works on every other browser/OS combination I tried. Are you able to swap it to ■ instead? Otherwise, this all looks good.